snap/snapcraft.yaml

```
@@ -0,0 +1,49 @@
+name: charmed-karapace
+base: core22
+version: '3.11.4'
+summary: Karapace on a snap
+description: |
+  This is a snap that bundles Karapace in order to be used in Charmed Operators, providing
+  automated operations management from day 0 to day 2 on Karapace registry solution, on top
+  of a Virtual Machine cluster and K8s cluster. It is an open source, end-to-end, production
+  ready data platform on top of cloud native technologies.
+
+confinement: strict
+
+system-usernames:
+  snap_daemon: shared
+
+# COS INTEGRATION
+# slots:
+#   logs:
+#     interface: content
+#     source:
+#       read:
+#         - $SNAP_COMMON/var/log/karapace
+
+apps:
+  daemon:
+    command: /opt/karapace/bin/start-wrapper.bash
```

uhm, maybe it should be 

```suggestion
    command: opt/karapace/bin/start-wrapper.bash
```

I was comparing this with the `mkpasswd` and the `charmed-kafka-snap`? Does this work anyway? Anyhow, it is a small point but maybe it is better to keep consistency

---

My guess it's that the snap only sees the internal filesystem when executing commands, unless given permission through plugs. But I agree it should have the same pattern.